sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -309,9 +307,9 @@ protected void addFields() {
 		addField(CaseDataDto.INVESTIGATION_STATUS, NullableOptionGroup.class);
 		addField(CaseDataDto.OUTCOME, NullableOptionGroup.class);
 		addField(CaseDataDto.SEQUELAE, NullableOptionGroup.class);
-		if (isConfiguredServer(CountryHelper.COUNTRY_CODE_GERMANY)) {
-			addField(CaseDataDto.REPORTING_TYPE);
-		}
+
+		addField(CaseDataDto.REPORTING_TYPE);
```

Comment:
Is it needed to make this available for all countries?

Reply:
It will be hidden for other countries because of the @HideForCountrysExcept annotation in CaseDtaDto.

The idea behind this annotation was to eliminate this kind of if statements.